pillar/edx/ansible_vars/cloud_deployment.sls

```diff
@@ -42,10 +42,11 @@
 {% set efs_id = 'fs-1f27ae56' %}
 {% elif environment == 'mitxpro-qa' %}
 {% set efs_id = 'fs-b3865653' %}
+{% set edxapp_google_analytics_account = 'UA-5145472-40' %}
 {% elif environment == 'mitxpro-production' %}
 {% set efs_id = 'fs-68918b88' %}
+{% set edxapp_google_analytics_account = 'UA-5145472-38' %}
 {% endif %}
-{% set edxapp_google_analytics_account = '' %}
```

Comment:
We should probably keep this in a final `else` block for the case where the environment doesn't match any of the listed cases.